src/test.rs

```diff
@@ -93,8 +93,8 @@ use futures::{future, FutureExt, TryFutureExt};
 #[cfg(feature = "websocket")]
 use futures::{SinkExt, StreamExt};
 #[cfg(feature = "websocket")]
-use tokio::{
-    sync::{mpsc, oneshot},
+use futures::{
+    channel::{mpsc, oneshot},
```

Comment:
what was the rational for this change?

Reply:
tokio's `UnboundedSender` doesn't implement `Sink` anymore and I couldn't quickly figure out how to rewrite this: https://github.com/seanmonstar/warp/pull/336/files/1786ff96aeb1d5bbb58ac93eae9594c4e6cd1489..6c95083a2783050622fcefa12e1ef0801f8f3e51#diff-c5fc4beef0f03493c2e9a9c876073f0fR514-R523

I kept running into weird errors where `read` wasn't a stream/future anymore...?

Reply:
ah, yeah you are right, it's simpler to just switch to std one